db/migrate/20170209203904_orchestration_stack_enhancements_for_ansible_service_provision.rb

```diff
@@ -0,0 +1,14 @@
+class OrchestrationStackEnhancementsForAnsibleServiceProvision < ActiveRecord::Migration[5.0]
+  def change
+    add_column :orchestration_stacks, :script_name, :string
+    add_column :orchestration_stacks, :repo_name,   :string
+    add_column :orchestration_stacks, :verbosity,   :string
+    add_column :orchestration_stacks, :start_time,  :timestamp
+    add_column :orchestration_stacks, :finish_time, :timestamp
+    add_column :orchestration_stacks, :userid,      :string
+    add_column :orchestration_stacks, :inventory,   :string
```

Comment:
In this context, it would be a list of hostnames.  I'll look into the Array column type that you suggested.